.github/workflows/testReports.yml

```diff
@@ -0,0 +1,41 @@
+name: .NET
+
+on:
+  push:
+    branches: [ master]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    environment: prod
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.x
+      - name: Restore dependencies
+        run: dotnet restore
+      - name: Remove Previous Result
+        run: rm -rf RingCentral.Tests/TestResults
+      - name: Code Coverage
+        run: dotnet test --collect:"XPlat Code Coverage"
+        env:
+          RINGCENTRAL_SERVER_URL: ${{ secrets.RINGCENTRAL_SERVER_URL }}
+          RINGCENTRAL_CLIENT_ID: ${{ secrets.RINGCENTRAL_CLIENT_ID }}
+          RINGCENTRAL_CLIENT_SECRET: ${{ secrets.RINGCENTRAL_CLIENT_SECRET }}
+          RINGCENTRAL_JWT_TOKEN: ${{ secrets.RINGCENTRAL_JWT_TOKEN }}
+          RINGCENTRAL_SENDER: ${{ secrets.RINGCENTRAL_SENDER }}
+          RINGCENTRAL_RECEIVER: ${{ secrets.RINGCENTRAL_RECEIVER }}
+      - name: Install ReportGenerator
+        run: dotnet tool install -g dotnet-reportgenerator-globaltool
+      - name: Execute Report Generation
+        run: reportgenerator "-reports:RingCentral.Tests/TestResults/*/coverage.cobertura.xml" "-targetdir:RingCentral.Tests/TestResults/coverage-report" -reporttypes:HtmlSummary
+      - name: View Report
+        uses: actions/upload-artifact@v4
+        with:
+          name: Reoprts
+          path: RingCentral.Tests/TestResults/coverage-report/
```

Comment:
Can we exclude the auto generated content from test coverage report?
Contents in these two folders are fully generated:

https://github.com/ringcentral/RingCentral.Net/tree/master/RingCentral.Net/Definitions
https://github.com/ringcentral/RingCentral.Net/tree/master/RingCentral.Net/Paths